packages/pds/src/bin.ts

```diff
@@ -20,6 +20,9 @@ const run = async () => {
   let blockstore: IpldStore
   let db: Database
 
+  const keypair = await crypto.EcdsaKeypair.create()
+  process.env.RECOVERY_KEY = keypair.did()
+
   const cfg = ServerConfig.readEnv()
```

Comment:
I've altered `readEnv()` so that it can receive overrides.  So I think this would also work:
```suggestion
  const keypair = await crypto.EcdsaKeypair.create()

  const cfg = ServerConfig.readEnv({ recoveryKey: keypair })
```

Reply:
oh nice that's much cleaner 👌